R/mrgsolve.R

```diff
@@ -259,7 +259,7 @@ mrgsim <-  function(x, data=NULL, idata=NULL, events=NULL, nid=1, ...) {
   x@args$data <- NULL
   x@args$events <- NULL
   
-  if(nid > 1) {
+  if(is.numeric(nid)) {
```

Comment:
The doc description is "[...] only used if idata and data are missing", so should the condition be updated to something like `!(have_data || have_idata) && is.numeric(nid)`?

Reply:
Thanks; I think we're consistent with the doc now.